src/App.js

```diff
@@ -7,18 +7,23 @@ import ArticlesList from "./components/ArticlesList";
 import SingleArticle from "./components/SingleArticle";
 
 function App() {
-  const [user, setUser] = useState({username: "grumpy19", name: "Paul Grump", avatar_url: "https://vignette.wikia.nocookie.net/mrmen/images/7/78/Mr-Grumpy-3A.PNG/revision/latest?cb=20170707233013"});
+  const [currUser, setCurrUser] = useState({
+    username: "grumpy19",
+    name: "Paul Grump",
+    avatar_url: "https://vignette.wikia.nocookie.net/mrmen/images/7/78/Mr-Grumpy-3A.PNG/revision/latest?cb=20170707233013"
+    });
+
```

Comment:
Could you add a login/logout - even just to sign you in as this user. Can then check that you can't delete any comments if not signed in